azure/src/main/scala/quasar/blobstore/azure/AzurePropsService.scala

```diff
@@ -21,24 +21,43 @@ import quasar.blobstore.services.PropsService
 
 import cats.data.Kleisli
 import cats.effect.{Async, ContextShift}
-import cats.syntax.applicative._
-import com.microsoft.azure.storage.blob.{BlobAccessConditions, BlobURL, ContainerURL}
-import com.microsoft.azure.storage.blob.models.BlobGetPropertiesResponse
-import com.microsoft.rest.v2.Context
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import com.azure.storage.blob.{BlobAsyncClient, BlobContainerAsyncClient}
+import com.azure.storage.blob.models.BlobProperties
+import scala.Option
+
 
 object AzurePropsService {
+
+  def fromBlobPropsArgs[F[_]: Async: ContextShift]//(
+      //blobClient: BlobAsyncClient,
+      // args: BlobPropsArgs)
+      : Kleisli[F, BlobPropsArgs, Option[BlobProperties]] = {
+    val res =
+      // Kleisli.pure(blobClient) andThen
+      // Kleisli.pure(args) andThen
+      // Kleisli[F, BlobAsyncClient, BlobPropsArgs](mkArgs(_).pure[F]) andThen
```

Comment:
Are these commented out lines here for a reason?

Reply:
@alissapajer No, this was the result of a hurried commit from me. I'll remove it